doc/source/whatsnew/v1.4.0.rst

```diff
@@ -973,6 +973,7 @@ Styler
 - Bug when combining non-sparse rendering and :meth:`.Styler.hide_columns` or :meth:`.Styler.hide_index` (:issue:`43464`)
 - Bug setting a table style when using multiple selectors in :class:`.Styler` (:issue:`44011`)
 - Bugs where row trimming and column trimming failed to reflect hidden rows (:issue:`43703`, :issue:`44247`)
+- Minor bug when attempting to apply styling functions to an empty DataFrame subset (:issue:`45313`)
```

Comment:
FWIW this behavior was a regression introduced after 1.3.

Reply:
not really, because the behaviour was somewhat undefined peviously. applying styles to an empty dataframe is an erroneous action and was only highlighted by your issue. just cause it didnt raise an error previously does not mean that was the intended behaviour.

Reply:
let's put in 1.5